zio-s3/src/main/scala/zio/s3/Live.scala

```diff
@@ -234,7 +234,8 @@ object Live {
   def connect[R](
     region: S3Region,
     provider: RIO[R with Scope, AwsCredentialsProvider],
-    uriEndpoint: Option[URI]
+    uriEndpoint: Option[URI],
+    forcePathStyle: Option[Boolean]
```

Comment:
use a boolean is enough :)

Reply:
Unfortunately false value is not the same as no value:

```java
// DefaultS3BaseClientBuilder.java

        if (serviceConfigBuilder.pathStyleAccessEnabled() != null) {
            Validate.validState(
                    clientContextParams.get(S3ClientContextParams.FORCE_PATH_STYLE) == null,
                    "ForcePathStyle has been configured on both S3Configuration and the client/global level. Please limit ForcePathStyle configuration to one location.");
        } else {
            serviceConfigBuilder.pathStyleAccessEnabled(clientContextParams.get(S3ClientContextParams.FORCE_PATH_STYLE));
        }
```